tests/test_Sequence.cc

```diff
@@ -29,81 +29,47 @@
 
 using namespace std::literals;
 using namespace task;
+using namespace task::literals;
 using Catch::Matchers::Contains;
 
-TEST_CASE("Sequence: Constructor without descriptive label", "[Sequence]")
+TEST_CASE("Sequence: Constructor", "[Sequence]")
 {
-    REQUIRE_THROWS_AS(Sequence{ "" }, Error);
-    REQUIRE_THROWS_AS(Sequence{ " \t\v\n\r\f" }, Error);
-}
-
-TEST_CASE("Sequence: Constructor with descriptive label", "[Sequence]")
-{
-    // label is to many characters -> throws Error
-    REQUIRE_THROWS_AS(Sequence{ std::string(Sequence::max_label_length + 1, 'c') },
-        Error);
-    // minimum label length with one character
-    REQUIRE_NOTHROW(Sequence{ "S" });
-    // label length with all characters filled
-    REQUIRE_NOTHROW(Sequence{ std::string(Sequence::max_label_length, 'c') });
-}
-
-TEST_CASE("Sequence: Constructor with get label", "[Sequence]")
-{
-    SECTION("label with leading blanks")
+    SECTION("No arguments")
     {
-        Sequence seq{ " test_sequence" };
-        REQUIRE(seq.get_label() == "test_sequence" );
+        Sequence sequence;
+        REQUIRE(sequence.empty());
+        REQUIRE(sequence.get_label() == "");
+        REQUIRE(sequence.get_name() == SequenceName{ "" });
     }
 
-    SECTION("label with trailing blanks")
+    SECTION("Label specified")
     {
-        Sequence seq{ "test_sequence " };
-        REQUIRE(seq.get_label() == "test_sequence" );
-    }
+        REQUIRE_THROWS_AS(Sequence{ " a\nb " }, Error);
```

Comment:
> All these tests here do not test `Sequence` code but `SequenceName` code. As that they are already more or less present in that test cases. 

That is not correct. I think it may have been in a previous iteration, but right now this test case checks the validation code for the human-readable *label*, not the machine-friendly *name*. There is no wrapper class for the label, so the checks are actually done in the `Sequence` constructor and in `set_label()`.